modules/oadp-release-notes-1-2-3.adoc

```diff
@@ -34,5 +34,10 @@ For a complete list of all issues resolved in the release of OADP 1.2.3, see the
 [id="known-issues-1-2-3_{context}"]
 == Known issues
 
-There are no known issues in the release of OADP 1.2.3.
+The {oadp-short} 1.2.3 has the following known issue:
 
+. Data Protection Application (DPA) does not reconcile when the credentials secret is updated
+
+Currently, the {oadp-short} Operator does not reconcile when you update the `cloud-credentials` secret. This occurs because there are no {oadp-short} specific labels or owner references on the `cloud-credentials` secret. If you create a `cloud-credentials` secret with incorrect credentials, such as empty data, the Operator reconciles and creates a Backup Storage Location (BSL) and registry deployment with the empty data. As a result, when you update  `cloud-credentials` secret with the correct credentials, the Operator does not immediately reconcile to catch the new credentials. As this issue is fixed in {oadp-short} 1.3,  the workaround is to update to {oadp-short} 1.3.
```

Comment:
I noticed you have a couple of extra spaces, @anarnold97. Also, the SSG has a very simple format for workarounds. :)

If you are online and want to fix this really quick, I'll pick up the merge review.

```suggestion
Currently, the {oadp-short} Operator does not reconcile when you update the `cloud-credentials` secret. This occurs because there are no {oadp-short} specific labels or owner references on the `cloud-credentials` secret. If you create a `cloud-credentials` secret with incorrect credentials, such as empty data, the Operator reconciles and creates a Backup Storage Location (BSL) and registry deployment with the empty data. As a result, when you update the `cloud-credentials` secret with the correct credentials, the Operator does not immediately reconcile to catch the new credentials. 

Workaround: Update to {oadp-short} 1.3.
```

Reply:
Sorry I missed the modifier _the_ on the first round. I was training a peer reviewer for the squad and got distracted.